tests/api/v1/globalconfig/post.js

```diff
@@ -41,17 +44,16 @@ describe(`api: POST ${path}`, () => {
       api.post('/v1/register')
       .set('Authorization', token)
       .send({
-        username: `${tu.namePrefix}test@test.com`,
-        email: `${tu.namePrefix}test@test.com`,
+        username: uname,
+        email: uname,
         password: 'abcdefghijklmnopqrstuvwxyz',
       })
       .end((err, res) => {
         if (err) {
           done(err);
-        } else {
-          testUserToken = res.body.token;
-          done();
```

Comment:
Wait, we never changes /v1/register api. Why do we need these changes /v1/register should still return a valid token. This is the token which is created on the fly, without db save.